From 2d51829665999d6e4e180b2702dd4ad7a0dc9aac Mon Sep 17 00:00:00 2001
From: Wayne Zou <b36644@freescale.com>
Date: Mon, 17 Sep 2012 15:48:15 +0800
Subject: ENGR00224513 V4L2 output: Avoid fb blank/unblank when open/close
 video device

Initialize release variable to avoid fb blank/unblank
when open/close video device only.

Signed-off-by: Wayne Zou <b36644@freescale.com>
---
 drivers/media/video/mxc/output/mxc_vout.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/video/mxc/output/mxc_vout.c b/drivers/media/video/mxc/output/mxc_vout.c
index 366d27a..367cc5f 100644
--- a/drivers/media/video/mxc/output/mxc_vout.c
+++ b/drivers/media/video/mxc/output/mxc_vout.c
@@ -861,6 +861,7 @@ static int mxc_vout_open(struct file *file)
 
 		vout->win_pos.x = 0;
 		vout->win_pos.y = 0;
+		vout->release = true;
 	}
 
 	file->private_data = vout;
-- 
1.7.9.5

